CHANGELOG.md

```diff
@@ -1,3 +1,286 @@
+# 1.8.0 (May 2025)
+
+_Changes since 1.7.3_
+
+## Highlights
+
+### Multiple Platforms
+
+- Compose Multiplatform codebase is fully migrated to K2. Please note that native and web klibs can be consumed only with Kotlin 2.1.0 or newer. Also, due to underlying changes in the compiler plugin, it's better to recompile libraries against the new version. Please let us know if you find any compatibility issues during this migration [#1778](https://github.com/JetBrains/compose-multiplatform-core/pull/1778)
+- Implement multiplatform `BackHandler` and `PredictiveBackHandler`. And use them in material3 widgets and androidx-navigation library [#1771](https://github.com/JetBrains/compose-multiplatform-core/pull/1771)
+
+### iOS
+
+- Remove experimental flag from `fun enableTraceOSLog()` [#1652](https://github.com/JetBrains/compose-multiplatform-core/pull/1652)
+
+### Web
+
+-  Improves text input support in Safari on  mobile and desktop [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
+-  Correct certain text input scenarios on Web targets
+New `WebTextInputService` seeks to achieve the following goals:
+   1. Add support to Safari (which happens to have its own flow of input events that are inconsistent with other browsers)
+   2. Correct behavior for composition and accent dialogue scenarios
+   3. Introduce some changes that we will use for improving accessibility
+- `skiko.js` is redundant in case of K/Wasm Compose Multiplatform for web applications and it can be removed from index.html files to not load redundant files. `skiko.js` will be removed from the k/wasm distribution in the future releases. `skiko.js` is still needed in case of K/JS Compose Multiplatform for web apps [#5134](https://github.com/JetBrains/compose-multiplatform/pull/5134)
+
+### Resources
+
+- Add FontVariation.Settings support to the resources library [#5183](https://github.com/JetBrains/compose-multiplatform/pull/5183)
+
+## Breaking Changes
```

Comment:
In the middle of 1.8 cycle we splitted `Breaking Changes` into `Breaking Changes` and `Migration Notes`. But didn't apply it retrospectively. We need to move it now.